cmd/coder/sync.go

```diff
@@ -29,6 +33,23 @@ func (cmd *syncCmd) RegisterFlags(fl *pflag.FlagSet) {
 	fl.BoolVarP(&cmd.init, "init", "i", false, "do initial transfer and exit")
 }
 
+// Returns local rsync protocol version as a string.
```

Comment:
Apologies for missing this earlier in the review. These comments should start with the function/method name. So this would be "version returns local rsync protocol version as a string."

(And same for the other `Version` method)

Reply:
No problem.  I appreciate you teaching me the style you prefer prior to the interview next week so if I get the job, I'm already style trained.